lib/helpers.py

```diff
@@ -440,6 +453,11 @@ def __init__(self, customer, revenue_event, chunk_size=10 ** 6):
 
         self.chunk_size = chunk_size
 
+        self.cache_folder = "cache-v{0}".format(__version__)
```

Comment:
`cache_folder` is only used in the method `read_csv`

Can you please remove it as a class property and move it back to the method?

Reply:
👍